libs/application/templates/passport/src/lib/dataSchema.ts

```diff
@@ -11,6 +11,18 @@ const isValidPhoneNumber = (phoneNumber: string) => {
   return phone && phone.isValid()
 }
 
+const guardian = z.object({
+  name: z.string().min(1),
+  nationalId: z.string().refine((x) => (x ? nationalIdRegex.test(x) : false)),
+  email: z
+    .string()
+    .refine((v) => isValidEmail(v), { params: error.invalidValue }),
```

Comment:
Zod has `z.string().email()`.
I assume `error.invalidValue` is just `ógilt gildi`, and if so; is this refine needed?